doc/nrf/ug_nrf5340.rst

```diff
@@ -56,10 +56,11 @@ Use this core for tasks that require high performance and for application-level
 The M33 TrustZone divides the application MCU into secure and non-secure domains.
 When the MCU boots, it always starts executing from the secure area.
 
-In Zephyr, the application core is divided into two different build targets:
+In Zephyr, the application core is built against any of the following build targets:
```

Comment:
What does it mean "build against" here? "for"? I see some entries on the web asking for clarification of this phrasing, so I would suggest simplifying it.

Reply:
`is built using one of the following build targets`?

Reply:
Yes, I also feel "is built using one of the following build targets" is correct. I have made the changes.
@greg-fer - Please check.